test/CMakeLists.txt

```diff
@@ -9,17 +9,36 @@
 set(CELERITAS_SOURCE_DIR "${PROJECT_SOURCE_DIR}")
 configure_file(gtest/detail/TestConfig.cc.in TestConfig.cc @ONLY)
 
-add_library(CeleritasTest
+set(CELER_SOURCES)
+list(APPEND CELER_SOURCES
   gtest/Test.cc
   gtest/detail/NonMasterResultPrinter.cc
   gtest/detail/ParallelHandler.cc
   gtest/detail/TestMain.cc
   gtest/detail/Utils.cc
   "${CMAKE_CURRENT_BINARY_DIR}/TestConfig.cc"
 )
+
+if(CELERITAS_USE_VecGeom)
+  list(APPEND CELER_SOURCES
+    geometry/GeoParamsTest.cc
+  )
+endif()
+
+add_library(CeleritasTest ${CELER_SOURCES})
 add_library(Celeritas::Test ALIAS CeleritasTest)
 target_link_libraries(CeleritasTest PUBLIC celeritas GTest::GTest)
 
+if(CELERITAS_USE_VecGeom)
+  if(CELERITAS_USE_CUDA)
```

Comment:
Simplify to `if(CELERITAS_USE_VecGeom AND CELERITAS_USE_CUDA)`

Reply:
Done.